### DISCUSSION TITLE
posts/obsidian-based-lightweight-paper-mgmt/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#24] [Announcements]
# posts/obsidian-based-lightweight-paper-mgmt/

基于Obsidian的高相关度文献关系梳理

https://blog.wohin.me/posts/obsidian-based-lightweight-paper-mgmt/

**Comment by MatheMatrix** [comment]
感觉可以用大模型帮助分析自己读过的文献的关系，自动加一下链接或者标签之类的

**Reply by brant-ruan** [reply]
是个不错的思路。不过我可能比较保守，不想在核心文献阅读上用大模型替代自己去处理。感觉这些步骤手动做一遍还是必要的，也不麻烦。
